extensions/simple_extension.go

```diff
@@ -69,6 +70,8 @@ func (EnumArg) toTypeString() string {
 	return "req"
 }
 
+func (v EnumArg) marker() {}
```

Comment:
marker method should be named for the kind of mark. E.g. argumentMarker() here.

Reply:
fixed it